src/components/subscriptions/LicenseActions.jsx

```diff
@@ -21,27 +42,32 @@ export default function LicenseAction({ user }) {
           }, {
             text: 'Revoke',
             // eslint-disable-next-line no-console
-            handleClick: () => { console.log('Revoke clicked for', user); },
+            handleClick: closeModal => (<LicenseRevokeModal
```

Comment:
Also, is it possible we can write this code from L46 to L51 at one place and use that at both of these places.